### PR TITLE
add debian:jessie console :)

### PR DESCRIPTION
**Pull request by imikushin**
add debian:jessie console :)

### CODE REVIEW

File: scripts/extraimages/01-debianconsole [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+COPY scripts/dockerimages/scripts/entry.sh /usr/sbin/
+COPY scripts/dockerimages/scripts/console.sh /usr/sbin/
+COPY scripts/dockerimages/scripts/update-ssh-keys /usr/bin/
+RUN echo 'RancherOS \\n \l' > /etc/issue
+RUN locale-gen en_US.UTF-8
+RUN addgroup --gid 1100 rancher && \
+    addgroup --gid 1101 docker && \
+    useradd -u 1100 -g rancher -G docker,sudo -m -s /bin/bash rancher && \
+    echo '## allow password less for rancher user' >> /etc/sudoers && \
+    echo 'rancher ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
```
Comment (wlan0) [inline scripts/extraimages/01-debianconsole]: @imikushin docker user needs to be added as well, https://github.com/rancherio/os/pull/239/files#diff-e14a520ce1af4fb35a5ef4f0578ebd65R8

Comment (imikushin) [inline scripts/extraimages/01-debianconsole]: Thanks @wlan0. 
Done.

**Comment by imikushin** [comment]
@ibuildthecloud I've rebased this PR on top of #239

**Comment by ibuildthecloud** [comment]
I haven't merged this because we have this chicken and egg problem between merging os and os-service if the image comes from os.  We really need to move all the of images out of rancherio/os into a separate project rancherio/os-images.  That way we can build new images and push them independent of a RancherOS release.

Anyhow, for this PR I'll merge right before we cut the release.

**Comment by imikushin** [comment]
Sounds great!

**Pull request merged**